web/controllers/post_controller.ex

```diff
@@ -38,7 +38,8 @@ defmodule Tilex.PostController do
            |> Repo.preload([:channel])
            |> Repo.preload([:developer])
 
-    render(conn, "show.html", post: post)
+    Plug.Conn.assign(conn, :canonical_url, "https://til.hashrocket.com" <> Tilex.Router.Helpers.post_path(conn, :show, post))
```

Comment:
You can use the function `assign/3` directly in controllers as they are imported via Phoenix.Controller. However I don't think we need`:canonical_url`